sources/engine/Stride.Rendering/Rendering/RenderContext.cs

```diff
@@ -89,7 +89,7 @@ internal RenderContext(IServiceRegistry services)
         public RenderSystem RenderSystem { get; set; }
 
         /// <summary>
-        /// The current visibility group from the <see cref="SceneInstance"/> and <see cref="RenderSystem"/>.
+        /// The current visibility group from the <see cref="Stride.Rendering.VisibilityGroup"/> and <see cref="RenderSystem"/>.
```

Comment:
Thanks, I will correct this.

Reply:
Ok. I have to reverse this back. The `SceneInstance` can't be resolved with the docfx unfortunatelly, because of some namespace conflicts. For example this `IAnimatableSpriteProvider.cs` has namespace `Stride.Engine` while it is placed in the `Stride.Rendering` project. So the cref is referencing local namespace in the `Stride.Engine` instead where `SceneInstance` is living. I will just remove the cref till one day this is fixed correctly.

<img width="1894" alt="image" src="https://github.com/user-attachments/assets/f1f3db63-e938-4a41-b209-92e8325e09bb">

Reply:
Ok. I decided to keep `cref` and added the full namespace, even it is not working. Once the namespace conflict is fixed, this cref will just start working 🙂